cmd/tools/grafana/grafana.go

```diff
@@ -26,9 +26,10 @@ import (
 )
 
 const (
-	clientTimeout      = 5
-	grafanaFolderTitle = "Harvest 2.0"
-	grafanaDataSource  = "Prometheus"
+	clientTimeout           = 5
+	grafanaFolderTitle      = "Harvest 2.0"
+	grafana7modeFolderTitle = "7 mode"
```

Comment:
would using the title `7-mode` simplify the code later that needs to remove the space?